### ISSUE TITLE
Configurable licence in footer

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#1544]
### Discussed in https://github.com/open-sdg/open-sdg/discussions/1538

<div type='discussions-op-text'>

<sup>Originally posted by **LucyGwilliamAdmin** January 17, 2022</sup>
UK are customising the site footer in order to be able to show a license (see bottom of any page on [UK platform](https://sdgdata.gov.uk/)).

A lot of websites require this so maybe we should consider adding it in to Open SDG?

Looking around at a few diff sites, where a site does have a license, they always have text but only sometimes have an image, so maybe image shouldn't be compulsory.

Here is the link to the GDS component on which the UK version is based (ignore Crown copyright bit):
https://design-system.service.gov.uk/components/footer/with-meta/index.html</div>

**Comment by brockfanning** [comment]
@LucyGwilliamAdmin While working on this a different approach occurred to me which might be worth considering. Instead of adding configuration settings for this, we could add a blank include file which is intended to be overridden, exactly like we do with the [scripts-custom.html file](https://github.com/open-sdg/open-sdg/blob/1.8.0-dev/_includes/scripts-custom.html). For this purpose, we would call it "footer-custom.html". I started considering this when I realized that this feature would also need a config setting for the alt text of the image, which means that it would need three settings total (image, alt text, text of license).

I admit that the config setting approach has the benefit of requiring less technical expertise (no HTML involved) but I also see some benefits to this blank include file approach:

1. More flexible (eg, can show multiple licenses, multiple images, and other things besides licenses)
2. This approach could be used in many places throughout the platform, so could become a standard way to customize things. Over time this could prevent the site configuration form from growing larger and larger (eg, avoids adding the three new config settings that this ticket would require).
3. Takes advantage of the Jekyll "override" workflow without the downsides of overriding Open SDG stuff
